Fixed comments by @tg-msft on APIView

NiveditJain opened the pull request:
APi View: https://apiview.dev/Assemblies/Review/4200a804f382479bb4d2bb4a3323b8c7

Commit:
fixed comments by @tg-msft

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.Developer.LoadTesting](https://apiview.dev/Assemblies/Review/a253ce842d8541bea40e60076b3895f3)

Commit:
exportedAPIs

Commit:
changed,
TestRunOperation -> TestRunResultOperation
FileUploadOperation -> FileUploadResultOperation

Commit:
added exported APIs
